mlir/utils/jenkins/Jenkinsfile

```diff
@@ -765,7 +776,9 @@ pipeline {
                                 sh 'date --utc +%Y-%m-%d > perf-run-date'
                                 // Run MLIR perf benchmarks
                                 sh 'python3 ./bin/perfRunner.py'
-                                sh 'python3 ./bin/perfRunner.py --op=gemm --batch_both --configs_file=../mlir/utils/performance/bert-configs'
+                                sh """python3 ./bin/perfRunner.py --op=gemm --batch_all \
+--configs_file=../mlir/utils/performance/bert-configs \
+--tuning_db=../mlir_tuning_${CHIP}.tsv"""
```

Comment:
I think it is better to use a path relative to ${WORKSPACE}.

Reply:
Good point